ctyparser/bigcty.py

```diff
@@ -172,7 +173,9 @@ def update(self) -> bool:
 
             with tempfile.TemporaryDirectory() as temp:
                 path = pathlib.PurePath(temp)
-                dl_url = f'http://www.country-files.com/bigcty/download/{update_date[:4]}/bigcty-{update_date}.zip'  # TODO: Issue #10
```

Comment:
It would probably be good to keep that URL as a fallback, like the other URL format already is